<?xml version="1.0"?>
<launch>
	<rosparam ns="color_range" command="load" file="$(find zeabus_vision)/params/color_top.yaml">
	</rosparam>
	<node pkg="zeabus_vision" name="color_range_top" type="color_range_top.py" output="screen">
		<param name="topic_top" value='/leftcam_top/image_raw/compressed'/>
	</node>
</launch>
